application/scss/components/_hmrc-internal-header.scss

```diff
@@ -42,3 +42,11 @@
     font-size: 24px;
   }
 }
+
+.hrmc-lang-select {
+  @include govuk-font(16);
+  float: right;
+  margin: 1rem 0;
+  width: auto;
+  list-style:none;
+}
```

Comment:
I don't think we need this now. Can you remove please?

Reply:
Done